.github/workflows/evaluation.yml

```
@@ -56,9 +56,9 @@ jobs:
       - name: Run LLVM
         continue-on-error: true
         run: |
-          (cd bv-evaluation; python3 ./compare-leansat-vs-bitwuzla-llvm.py -j48)
+          (cd bv-evaluation; python3 ./compare.py instcombine-j48)
```

```suggestion
          (cd bv-evaluation; python3 ./compare.py instcombine -j48)
```

Does this need a whitespace here?